pkg/virt-controller/watch/migration.go

```diff
@@ -1081,6 +1115,15 @@ func (c *MigrationController) handlePendingPodTimeout(migration *virtv1.VirtualM
 	return nil
 }
 
+func targetPodDisappeared(pod *k8sv1.Pod, vmi *virtv1.VirtualMachineInstance) bool {
```

Comment:
I am not sure this is the most accurate name. I think this condition does 2 things: `targetPodDisappeared` and `migrationDidNotStart`

@vladikr WDYT?

Reply:
`targetPodDisappeared' is a bit implicit in saying that there was a problem with the migration, but if you have a better name sure thing!